jellyfin-core/src/commonMain/kotlin/org/jellyfin/sdk/discovery/RecommendedServerDiscovery.kt

```diff
@@ -46,10 +48,28 @@ public class RecommendedServerDiscovery constructor(
 		val issues = mutableSetOf<RecommendedServerIssue>()
 		val scores = mutableSetOf<RecommendedServerInfoScore>()
 
+		// Failure checks
+		if (result.systemInfo.isFailure) {
```

Comment:
Should be part of the when statement for system info checks.

Reply:
I looked into the Result class and it's api. Instead of checking if it's null or it's a failure - I just check if there is an exception (implies that there is a failure as per the api).

I also moved the missing system info check to the above let block, otherwise we will have both missing system info issue recommendation and ssl recomendation as well. Here in this case, we are defaulting to MissingSystemInfo issue recomendation when the server replies with something we do not expect (the rest of the known failures line 136) and all other unknown errors.

What do you think?